app/controllers/grades_controller.rb

```
@@ -361,17 +376,76 @@ def get_heatgrid_data_for(maps)
+                # Inject SectionHeader sentinels so the frontend can render section headings
+                aq = AssignmentQuestionnaire.where(assignment_id: @assignment.id, used_in_round: round).first
+                inject_section_headers(reviewee_scores[round_symbol], aq&.questionnaire_id)
+            end
+
+        else
+            # Single-round assignment: the original code only had the varying_rubrics_by_round?
+            # branch and returned nil for everything else, so no scores were ever shown for
+            # standard (non-varying) assignments. This else branch collects all submitted
+            # responses and groups them under a synthetic "Round 1" key, matching the same
+            # data shape that the varying-round branch produces.
+            round_symbol = ("#{maps.first.questionnaire_type}-Round-1").to_sym
```

RuboCop (1.87.0)

[convention] Use 2 (not 4) spaces for indentation.

(Layout/IndentationWidth)